.github/workflows/dev-image-update.yaml

```diff
@@ -20,22 +20,31 @@ jobs:
           repository: 'broadinstitute/datarepo-helm-definitions'
           token: ${{ secrets.HELM_REPO_TOKEN }}
           path: datarepo-helm-definitions
+      - name: "Bump the tag to a new version"
+        uses: broadinstitute/datarepo-actions@0.8.1
```

Comment:
Next tagged release is gonna be 0.7.0 unless you changed https://github.com/broadinstitute/datarepo-actions/blob/master/.github/workflows/tag.yaml